File: app/api/connected/connected.py
import requests  # pip3 install requests

url = "http://localhost:8000/api/v1/connected?user=call-me"

authorization = "call_me_api_key_secret"

headers = {
    'Authorization': authorization,
    'Content-Type': 'application/json'
}

response = requests.get(url, headers=headers)

print(response.json())

File: app/api/connected/connected.js
'use strict';

const url = 'http://localhost:8000/api/v1/connected?user=call-me';

const authorization = 'call_me_api_key_secret';

fetch(url, {
    method: 'GET',
    headers: {
        Authorization: authorization,
        'Content-Type': 'application/json',
    },
})
    .then((response) => response.json())
    .then((data) => console.log(data))
    .catch((error) => console.error('Error:', error));
